ProcessMaker/Models/ProcessRequestToken.php

```diff
@@ -708,6 +709,57 @@ public function valueAliasTask($value, $expression)
         };
     }
 
+    /**
+     * PMQL value alias for the case number field related to process request
+     *
+     * @param string value
+     * @param ProcessMaker\Query\Expression expression
+     *
+     * @return callable
+     */
+    public function valueAliasCase_Number(string $value, Expression $expression): callable
+    {
+        return function ($query) use ($expression, $value) {
+            $query->whereHas('processRequest', function ($query) use ($expression, $value) {
+                return $query->where('case_number', $expression->operator, $value);
+            });
+        };
+    }
+
+    /**
+     * PMQL value alias for the case title field related to process request
+     *
+     * @param string value
+     * @param ProcessMaker\Query\Expression expression
+     *
+     * @return callable
+     */
+    public function valueAliasCase_Title(string $value, Expression $expression): callable
+    {
+        return function ($query) use ($expression, $value) {
+            $query->whereHas('processRequest', function ($query) use ($expression, $value) {
+                $query->where('case_title', $expression->operator, $value);
+            });
+        };
+    }
+
+    /**
+     * PMQL value alias for the process name field related to process request
+     *
+     * @param string value
+     * @param ProcessMaker\Query\Expression expression
+     *
+     * @return callable
+     */
+    public function valueAliasName(string $value, Expression $expression): callable
```

Comment:
Seems this should be `valueAliasProcess_Name` so the user could filter by process_name:
```
process_name like "Process Name"
```

```suggestion
    public function valueAliasProcess_Name(string $value, Expression $expression): callable
```

Reply:
I was fixed